count_ignores.sh

```diff
@@ -0,0 +1,40 @@
+current_dir=$(pwd)
+
+cd exercises
+
+for exercise_name in *
+do
+
+#Get filename
+file_name="$exercise_name/tests/*.rs"
+
+#Commented tests
+comment_test_num=$(cat $file_name | grep "// \#\[test\]" | wc -l)
```

Comment:
It is probably preferable to `grep PATTERN FILE` rather than `cat FILE | grep PATTERN`, right? Is there a difference between the two?